spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe Users::SessionsController, type: :controller do
+  describe "POST #create" do
+    before :each do
+      @request.env["devise.mapping"] = Devise.mappings[:user]
+      @user = FactoryGirl.create :user
```

Comment:
let(:user) { create(:user) }, after opening the spec declaration 
https://www.relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let